### PR TITLE
Add skeletal profile page css and html

### PR DESCRIPTION
**Pull request by sally**

### CODE REVIEW

File: app/assets/stylesheets/profile_page.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.user-info {
+  height: 500px;
+}
+
+.user-picture-container{
+  background-color: red;
+}
+
+.user-kids-container{
+  background-color: yellow;
```
Comment (sleiken) [inline app/assets/stylesheets/profile_page.scss:31]: good job! @parkyngj

File: app/assets/stylesheets/profile_page.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+}
+
+.user-kids-info-container:after, .user-kids-info-container:before {
+  right: 100%;
+  top: 50%;
+  border: solid transparent;
+  content: " ";
+  height: 0;
+  width: 0;
```
Comment (sleiken) [inline app/assets/stylesheets/profile_page.scss:46]: wow! nice job!! @parkyngj

File: app/assets/stylesheets/profile_page.scss (hunk 1 above)
Comment (sally) [reply app/assets/stylesheets/profile_page.scss:31]: ｗｈａｔ

File: app/assets/stylesheets/profile_page.scss (hunk 2 above)
Comment (sally) [reply app/assets/stylesheets/profile_page.scss:46]: (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧

**Pull request merged**